java/lang/os_command_injection.yml

```
@@ -0,0 +1,65 @@
+imports:
+  - java_shared_lang_instance
+patterns:
+  - pattern: |
+      $<RUNTIME_INIT>.exec($<DYNAMIC_INPUT>);
+    filters:
+      - variable: RUNTIME_INIT
+        detection: java_lang_os_command_injection_runtime
+      - variable: DYNAMIC_INPUT
+        detection: java_lang_os_command_injection_dynamic_input
+  - pattern: |
+      $<PROCESS_BUILDER_INIT> $<_> = new $<PROCESS_BUILDER_INIT>($<DYNAMIC_INPUT>);
+    filters:
+      - variable: PROCESS_BUILDER_INIT
+        detection: java_lang_os_command_injection_process_builder
+      - variable: DYNAMIC_INPUT
+        detection: java_lang_os_command_injection_dynamic_input
+  - pattern: |
+      $<PROCESS_BUILDER_INIT>.command($<DYNAMIC_INPUT>);
+    filters:
+      - variable: PROCESS_BUILDER_INIT
+        detection: java_lang_os_command_injection_process_builder
+      - variable: DYNAMIC_INPUT
+        detection: java_lang_os_command_injection_dynamic_input
+auxiliary:
+  - id: java_lang_os_command_injection_runtime
+    patterns:
+      - pattern: $<JAVA_LANG_RUNTIME_INIT>;
+        filters:
+          - variable: JAVA_LANG_RUNTIME_INIT
+            regex: \A(java\.lang\.)?Runtime
+  - id: java_lang_os_command_injection_process_builder
+    patterns:
+      - pattern: $<JAVA_LANG_PROCESS_BUILDER_INIT>;
+        filters:
+          - variable: JAVA_LANG_PROCESS_BUILDER_INIT
+            regex: \A(java\.lang\.)?ProcessBuilder
+  - id: java_lang_os_command_injection_dynamic_input
+    patterns:
+      - pattern: $<...>$<USER_INPUT>$<...>;
```

Does it work without the ellipsis? I think it should, as it's a single node pattern.

```suggestion
      - pattern: $<USER_INPUT>;
```